src/pages/Appointments/AppointmentDetail.tsx

```
@@ -188,18 +188,95 @@ export default function AppointmentDetail(props: Props) {
+          <PatientInfoCard
+            patient={appointment.patient}
+            facilityId={facilityId}
+            tags={appointment.tags}
+            onTagsUpdate={() => {
+              queryClient.invalidateQueries({
+                queryKey: ["appointment", appointment.id],
+              });
+            }}
+            tagEntityType="appointment"
+            tagEntityId={appointment.id}
+            children={
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)